go/src/koding/klient/machine/mount/index/kite.go

```diff
@@ -0,0 +1,55 @@
+package index
+
+import (
+	"github.com/koding/kite"
+)
+
+// KiteHandlerHead creates a kite handler function that, when called, invokes
+// index package Head method.
+func KiteHandlerHead() kite.HandlerFunc {
```

Comment:
It would be cleaner and I'm aware of this. However, I am planning to go even further and shorten other [kite.go](https://github.com/koding/koding/blob/master/go/src/koding/klient/machine/machinegroup/kite.go) file with the one generic handler "factory" for all functions.

Code coverage will make such changes more beneficial.